docs/DeveloperGuide.md

```diff
@@ -293,69 +293,95 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
       
       
-**UC01 - Add a visit using Date, Location, Person**
+**UC01 - Add a visit using PersonId, LocationId, Date**
 
 **MSS**
 
 1. User chooses to add a visit to the visits history list.
 2. VirusTracker requests for details of the visit in a specific format. 
-3. User enters the required details (Person, Date, Location).
+3. User enters the required details (PersonId, LocationId, Date).
```

Comment:
I think for this, it could just be "User enters the required details" since implementation/UI details are not needed for use cases.